automation/build-bin.ts

```diff
@@ -343,6 +343,20 @@ async function signWindowsInstaller() {
 	}
 }
 
+/**
+ * Wait for Apple Installer Notarization to continue
+ */
+async function afterSignHook(): Promise<void> {
```

Comment:
No need to make this change straight away, but I think this function's name would be clearer as `notarizeMacInstaller()`.